examples/check.py

```diff
@@ -47,7 +47,23 @@
 print "======================================================="
 print "\n"
 
-# Creating a Check using the previously created bank account
+# Verifying a Bank Account with the microdeposit amounts
```

Comment:
It errors

On Mon, Apr 20, 2015 at 12:38 PM, Leore Avidar <notifications@github.com>
wrote:

>> @@ -47,7 +47,23 @@
>>  print "======================================================="
>>  print "\n"
>>  
>> -# Creating a Check using the previously created bank account
>> +# Verifying a Bank Account with the microdeposit amounts
> what happens when you try to verify an account that has already been verified?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lob/lob-python/pull/64/files#r28721371